src/main/java/com/quirko/logic/bricks/RandomBrickGenerator.java

```diff
@@ -12,15 +12,19 @@
 
     private final Deque<Brick> nextBricks = new ArrayDeque<>();
 
-    public RandomBrickGenerator() {
+    public RandomBrickGenerator(String diff) {
         brickList = new ArrayList<>();
         brickList.add(new IBrick());
         brickList.add(new JBrick());
         brickList.add(new LBrick());
         brickList.add(new OBrick());
-        brickList.add(new SBrick());
-        brickList.add(new TBrick());
-        brickList.add(new ZBrick());
+        if(!diff.equalsIgnoreCase("easy")){

```

Comment:
I am not sure if I like this design. So these bricks is never seen unless the difficulty is easy.

Please use some other approach instead of completely removing some brick options.